samples/src/test/java/org/autorefactor/refactoring/rules/samples_out/SwitchSample.java

```diff
@@ -361,4 +361,169 @@ public void doNotReplaceWithSwitchOnPrimitiveWrapper(Integer i1) {
             i = 30;
         }
     }
+
+    public void mergeCases(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 0:
+        case 1:
+            i = 0;
+            break;
+        default:
+            i = 1;
+            break;
+        }
+    }
+
+    public void mergeCasesWithUnbrokenCase(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 1:
+            i = 1;
+        case 0:
```

Comment:
I was not clear enough.
I actually think it would be better if they were merged.

I can only think of exceptional cases where someone would want to be explicit by mentioning both `case X` and `default`, but I think they are seldom.

Please merge them if you can do it.